.github/workflows/validate_translation.yml

```diff
@@ -0,0 +1,47 @@
+name: Validate ASVS Translation
+
+on:
+  workflow_dispatch:
+    inputs:
+      translated_folder:
+        description: "Name of the folder containing translated files (inside docs/)"
+        required: true
+
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+
+      - name: Run validation scripts
+        run: |
+          set -e
```

Comment:
I'd just move all of this into a script (eg. `5.0/tools/translation_validation/check_translation.sh`) and have here:

~~~sh
5.0/tools/translation_validation/check_translation.sh "${{ github.event.inputs.translated_folder }}"
~~~